### DISCUSSION TITLE
HW2-notation and assumption

### DISCUSSION THREAD

**Discussion by seanie12** [seominjoon/kaist-ai605#24] [Q&A] [answered]
I am bit confused about the notation, Var(QK/sqrt{d_k}) = 1.

Does the 1 denote the matrix of which all the elements one?

And do we assume that all the random variables q_i, k_i (row vector of Q,K) are mutually independent?

Thank you.

**Comment by seominjoon** [comment, marked as answer]
Yes, you are right. The notation was a bit lazy..  The proof is about whether every element of whatever goes in the Var() function is 1.
Yes, please assume that all random variables in Q and K are mutually independent.

**Reply by seanie12** [reply]
Thank you.
